### ISSUE TITLE
Playlists support 

### ISSUE THREAD

**Issue by itsWindows11** [huynhsontung/Screenbox#110]
It would be good to be able to create playlists with a custom cover/banner and add songs/videos to them, and also add them to play queue.

**Comment by zohxin** [comment]
yeah please add this, otherwise this 100% replaces window's media player and VLC

**Comment by dayanidhi07** [comment]
Yes Guys, If playlist option is added it would be the best media player for windows in the market.

**Comment by dayanidhi07** [comment]
Hi team, is this feature in development, or is there any update on this?
